docs/src/content/docs/sdk.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Ultravox SDK"
+sidebar:
+  order: 0
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import SDKCards from '../../components/SDKCards.astro';
+
+The Ultravox [REST API](./api/auth/) is used to create calls but you must use one of the Ultravox client SDKs to join and end calls. This page uses examples in JavaScript. The concepts are the same across all the [SDK flavors](#sdk-flavors).
```

Comment:
nit:  I'd say "SDKs" or "SDK implementations" instead of "flavors."  Build "flavors" have specific meaning especially in the Android world.

Reply:
ok will clean that up.